src/components/CallView/shared/LocalVideo.vue

```diff
@@ -21,7 +21,8 @@
 <template>
 	<div id="localVideoContainer"
 		class="videoContainer videoView"
-		:class="{ speaking: localMediaModel.attributes.speaking, 'video-container-grid': isGrid, 'video-container-stripe': isStripe }">
+		:class="videoContainerClass"
+		:aria-label="videoContainerAriaLabel">
```

Comment:
I think this should go?
`Bad sent audio quality.` is now always added as aria label on the local video?

Reply:
[The aria label is added only if the connection is bad](https://github.com/nextcloud/spreed/pull/3839/files#diff-1c64a9150b5f78204e616221a463fe5cR129), as in that case the video is dimmed, but as that is a purely visual information I thought that an aria label would be good.

Although now that I see it it should be `if (!this.showQualityWarning) {` instead of `if (!this.senderConnectionQualityIsBad) {`...